js/script.js

```diff
@@ -25,4 +25,206 @@ menuClose.addEventListener('click', closeMenu);
+    sourceLink: { link: 'https://github.com/lenkon/Portfolio' },
+    buttonText: 'See Project',
+    id: 3,
+  },
+
+  {
+    title: 'Uber Navigation',
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    image: './images/large/Snapshoot Portfolio - Uber Navigation.png',
+    alt: 'uber navigation project image',
+    companyRoles: ['Uber', 'Lead Developer', '2018'],
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    liveLink: { link: './index.html' },
+    sourceLink: { link: 'https://github.com/lenkon/Portfolio' },
+    buttonText: 'See Project',
+    flip: 1,
+    id: 4,
+  },
+];
+
+const createProjectDetails = (item) => {
+  const card = document.createElement('div');
+  if (item.flip === 1) {
+    card.classList.add('card-works-container', 'card-interchange');
+  } else {
+    card.classList.add('card-works-container');
+  }
+  const roles = item.companyRoles.map((value, index) => {
+    if (index !== 0) {
+      return `<div class="card-bullet-separator"></div> <span class="card-canopy card-canopy-details">${value}</span>`;
+    }
+    return `<span class="card-canopy">${value}</span>`;
+  });
+
+  const skillsList = item.skills.map(
+    (count) => `<li class="card-tools">${count}</li>`,
+  );
+
+  const innerHtml = `
+    <img src="${item.image}" alt="${item.alt}">
+    <div class="card-sizing">
+      <h2 class="card-title-text">${item.title}</h2>
+      <div class="card-list">
+        ${roles.join('')}
+      </div>
+      <p class="card-main-text">${item.description}</p>
+      <ul class="card-tools-list">
+        ${skillsList.join('')}
+      </ul>
+      <div class="card-button-box">
+        <button card-id=${
+  item.id
+} class="card-button popup-card-button">See Project</button>
+      </div>
+    </div>
+  `;
+  card.innerHTML = innerHtml;
+  cardWorksWrapper.appendChild(card);
+};
+
+projectDetails.forEach((item) => createProjectDetails(item));
+
+const popupDetails = (item) => {
+  const roles = item.companyRoles.map((value, index) => {
+    if (index !== 0) {
+      return `<div class="card-bullet-separator"></div> <span class="card-canopy card-canopy-details">${value}</span>`;
+    }
+    return `<span class="card-canopy">${value}</span>`;
+  });
+
+  const skillsList = item.skills.map(
+    (count) => `<li class="card-tools">${count}</li>`,
+  );
+
+  const innerHtml = `<div id="popup-card-container"></div>
+  <section id="popup-card-section">
+    <article class="popup-card">
+      <header class="popup-details-header">
+        <nav class="popup-card-details">
+          <h2 class="card-title-text">${item.title}</h2>
+          <button id="popup-close-button" class="popup-close-button">
+            <img src="./images/Popup-close-Icon.png" alt="popup close icon">
+          </button>
+        </nav>
+        <div class="card-list">
+          ${roles.join('')}
+        </div>
+      </header>
```

Comment:
- [ ] You did a great job on the styles but for this project, you need to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z2va5AFebQpvwsk5-0) design as much as possible. For desktop screens, Please make sure to increase the font size for the project title, and tags and use the blur effect in the model background.

### Your design

![image](https://user-images.githubusercontent.com/83803414/210591653-ad6499fc-b9b4-4e8d-9eb8-fb0a5d2b25ea.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/210591711-5f41d16c-4762-4b5f-bb87-a383ce8cc56d.png)

Reply:
Hi @thecodechaser, thanks for the valuable review. I have made the required changes in the modal background and font sizes. Please go through it and let me know if you need any changes.